packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.test.tsx

```diff
@@ -377,6 +377,28 @@ describe('Autocomplete', () => {
     expectMenuToBeClosed();
   });
 
+  it('displays a custom error message', () => {
+    renderAutocomplete({ 'error-message': 'Something went wrong' });
+    expect(screen.getByText('Something went wrong')).toBeInTheDocument();
+  });
+
+  it('sets the display location of the error message', () => {
```

Comment:
Based on feedback around the error placement defaults, it might be nice to add test coverage so that it's clear the [healthcare theme](https://github.com/CMSgov/design-system/blob/jryan/wnmgds-2992-ds-autocomplete-pass-through-props-to-text-field/packages/design-system/src/components/config.ts#L68) should default to "bottom" and all the other [themes](https://github.com/CMSgov/design-system/blob/jryan/wnmgds-2992-ds-autocomplete-pass-through-props-to-text-field/packages/design-system/src/components/config.ts#L55) default to "top".